source/NetworkInformation/NetworkInterface.cs

```diff
@@ -418,7 +423,7 @@ public void RenewDhcpLease()
         {
             try
             {
-                UpdateConfiguration(UPDATE_FLAGS_DHCP_RELEASE | UPDATE_FLAGS_DHCP_RENEW);
+                UpdateConfiguration((int)UpdateOperation.DhcpRenew);
```

Comment:
@AdrianSoundy  this update is mirrored at the native end. Please check with ESP32 if it's working as expected.